### PR TITLE
Dev/jadelaga/vsix sign tool

### PR DESCRIPTION
**Pull request by javierdlg**
Updates the documentation for signing VSIX packages using `dotnet sign` and adds a CLI reference page. Another page is being added next to the sign code under [Dotnet/Sign](https://github.com/dotnet/sign)

### CODE REVIEW

**Comment by Court72** [comment]
Hi @javierdlg,

Microsoft authors and contributors need to make content contributions through the private repository so the content can be staged and validated by the current validation rules.

After you make your changes in the private repo, review the articles on staging, and fix any validation issues, you can add the sign-off comment to let PR reviewers know that the updates are ready to be merged. 
 
Read about [authors working in the public repo](https://review.docs.microsoft.com/help/contribute/contribute-authors-in-public-repo?branch=main) for more information. 

If you need help getting set up to work in the private repo, see [Get started contributing to the Microsoft Learn platform](https://review.learn.microsoft.com/help/get-started/?branch=main&tabs=learn).

We'll close this PR. Thank you.

File: docs/extensibility/signing-vsix-packages.md [hunk 1]
```diff
@@ -18,27 +18,57 @@ ms.subservice: extensibility-integration
...
+#### Installing Dotnet Sign
+1. Open a [Developer PowerShell](https://learn.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell?view=vs-2022) instance.
 
-2. Right click on the project node in Solution Explorer, selecting **Add &#124; Manage NuGet Packages**.  For more information on NuGet and adding NuGet packages see the [NuGet documentation](/NuGet) and [Package Manager UI](/NuGet/Tools/Package-Manager-UI) topics.
+1. Verify nuget.org is added and enabled as a NuGet source.
+    - Check your sources using `dotnet nuget list source` 
+    - Add NuGet.org as a source using `dotnet nuget add source -n NuGet.org https://api.nuget.org/v3/index.json`
 
-3. Search for VSIXSignTool from VisualStudioExtensibility and install the NuGet package.
+1. Install Sign by running `dotnet tool install sign --version <version> --global`, where `<version>` is the latest available version under [Sign (nuget.org)](https://www.nuget.org/packages/sign)
```
Comment (clairernovotny) [inline docs/extensibility/signing-vsix-packages.md:41]: Install Sign CLI by running...

File: docs/extensibility/signing-vsix-packages.md [hunk 2]
```diff
@@ -18,27 +18,57 @@ ms.subservice: extensibility-integration
...
+    - `--global` is optional but will allow using sign outside of the PowerShell instance.
 
-4. You can now run the VSIXSignTool from the project's local packages location. Consult the tool's command line help for your signing scenario (VSIXSignTool.exe /?).
 
-   For example to sign with a password protected certificate file:
+#### Using Dotnet Sign
```
Comment (clairernovotny) [inline docs/extensibility/signing-vsix-packages.md:45]: Using Sign CLI

File: docs/extensibility/signing-vsix-packages.md [hunk 3]
```diff
@@ -18,27 +18,57 @@ ms.subservice: extensibility-integration
...
+Once installed, Sign can be accessed in a Developer Powershell instance using `sign code <command> <options>`. For a breakdown of the options see
```
Comment (clairernovotny) [inline docs/extensibility/signing-vsix-packages.md:46]: Once installed, Sign CLI can be accessed...

File: docs/extensibility/signing-vsix-packages.md [hunk 4]
```diff
@@ -18,27 +18,57 @@ ms.subservice: extensibility-integration
...
+code certificate-store -s f5ec6169345347a7cd2f83af662970d5d0bfc914 -csp "eToken Base Cryptographic Provider" -k "NuGet Signing.629c9149345347cd2f83af6f5ec70d5d0a7bf616" -d "VSIX Signature" -u "http://timestamp.acs.microsoft.com/" "C:\Users\Contoso\Downloads\FingerSnapper2022.vsix"
+```
+
+#### Signing using PFX files
 
-   VSIXSignTool.exe sign /f \<certfile> /p \<password> \<VSIXfile>
 
 ## Related content
 - [Shipping Visual Studio Extensions](../extensibility/shipping-visual-studio-extensions.md)
+- [Dotnet Sign CLI Reference for VSIX Packages](../extensibility/dotnet-sign-CLI-reference-vsix.md)
```
Comment (clairernovotny) [inline docs/extensibility/signing-vsix-packages.md:74]: `[Sign CLI Reference for ...`

**Review by clairernovotny — COMMENTED** [review]
Please use Sign CLI everywhere the normal name of tool is referenced (as opposed to the actual CLI command in samples)

**Pull request closed without merging**